c/ldv-consumption/32_7a_cilled_linux-3.8-rc1-32_7a-drivers--gpu--drm--ttm--ttm.ko-ldv_main5_sequence_infinite_withcheck_stateful.cil.out.c

```diff
@@ -10461,18 +10461,22 @@ int main(void)
   int tmp___0 ;
 
   var_group1 = ldv_init_zalloc(sizeof(struct vm_area_struct));
-  var_group1->vm_private_data = __VERIFIER_nondet_pointer();
-  __VERIFIER_assume(var_group1->vm_private_data != ((void *)0));
+  var_group1->vm_private_data = malloc(sizeof(struct ttm_buffer_object));
   bo = (struct ttm_buffer_object *)var_group1->vm_private_data;
-  bo->bdev = __VERIFIER_nondet_pointer();
-  __VERIFIER_assume(bo->bdev != ((void *)0));
+  bo->bdev = bdev = (struct ttm_bo_device *)malloc(sizeof(struct ttm_bo_device));
```

Comment:
Nit pick: why are you doing an explicit type cast here? You didn't do so above, so it seems inconsistent. (Either way seems fine, there is no impact on the semantics, it's just inconsistent and hence surprising.)

Reply:
Right, good catch. I fixed this.